README.md

```diff
@@ -539,7 +500,7 @@ In either case:
 
 ## Syntax
 
-* <a name="no-for"></a>Never use `for`, unless you know exactly why. Most of the 
+* <a name="no-for"></a>Never use `for`, unless you know exactly why. Most of the
```

Comment:
i think its good to have in the doc at least to know the difference between for and each